target/linux/ath79/image/generic.mk

```diff
@@ -448,6 +459,32 @@ define Device/devolo_magic-2-wifi
 endef
 TARGET_DEVICES += devolo_magic-2-wifi
 
+define Device/dlink_dap-13xx
+  SOC := qca9533
+  DEVICE_VENDOR := D-Link
+  DEVICE_VARIANT := A1
```

Comment:
I'd prefer the variant at the device nodes.

Reply:
Also thought about that, and agree to prefer it over reducing redundancy here.